source/materials/OpticalMaterialProperties.cc

```diff
@@ -42,99 +42,86 @@ G4MaterialPropertiesTable* OpticalMaterialProperties::Epoxy()
 
   G4MaterialPropertiesTable* mpt = new G4MaterialPropertiesTable();
 
-  G4double energy[2] =
-    {1*eV, 10.751*eV};
-  G4double rindex[2] =
-    {1.54, 1.54};
-
-  // ABSORPTION LENGTH /////////////////////////////////////////////////////////
+  G4double energy[2] = {optPhotMinE_, optPhotMaxE_};
+  G4double rindex[2] = {1.54, 1.54};
 
+  // ABSORPTION LENGTH
   const G4int abs_entries = 17;
 
   G4double abs_energy[abs_entries]=
-    {1.*eV, 2.132*eV, 2.735*eV, 2.908*eV, 3.119*eV,
+    {optPhotMinE_, 2.132*eV, 2.735*eV, 2.908*eV, 3.119*eV,
      3.320*eV, 3.476*eV, 3.588*eV, 3.749*eV, 3.869*eV,
      3.973*eV, 4.120*eV, 4.224*eV, 4.320*eV, 4.420*eV,
-     5.018*eV, 8.*eV};
-  G4double abslength[abs_entries] =
+     5.018*eV, optPhotMaxE_};
+  G4double abs_length[abs_entries] =
     {15000.*cm, 326.*mm, 117.68*mm, 85.89*mm, 50.93*mm,
      31.25*mm, 17.19*mm, 10.46*mm, 5.26*mm, 3.77*mm,
      2.69*mm, 1.94*mm, 1.94*mm, 1.94*mm, 1.94*mm,
      1.94*mm, 1.94*mm};
 
   mpt->AddProperty("RINDEX", energy, rindex, 2);
-  mpt->AddProperty("ABSLENGTH", abs_energy, abslength, abs_entries);
+  mpt->AddProperty("ABSLENGTH", abs_energy, abs_length, abs_entries);
 
   return mpt;
 }
 
 G4MaterialPropertiesTable* OpticalMaterialProperties::EpoxyFixedRefr(G4double n)
 {
-  // Optical properties adjusted in order not to modify the trajectories of photons and
-  // not to absorb them before they meet the photocathode. To be set to real properties when available
+  // Optical properties adjusted in order not to absorb photons
+  // before they meet the photocathode.
```

Comment:
The choice of the word is not great, it should mean "active area". However, I've changed the description, since it doesn't reflect the current implementation of that property.